translations/base-ind.yaml

```diff
@@ -489,123 +489,121 @@ buildings:
+            description: Menggabungkan dua sabuk konveyor menjadi satu.
         splitter:
-            name: Splitter (compact)
-            description: Splits one conveyor belt into two.
+            name: Pemisah (Kompak)
+            description: Memisahkan satu sabuk konveyor menjadi dua.
         splitter-inverse:
-            name: Splitter (compact)
-            description: Splits one conveyor belt into two.
+            name: Pemisah (Kompak)
+            description: Memisahkan satu sabuk konveyor menjadi dua.
     storage:
         default:
-            name: Storage
-            description: Stores excess items, up to a given capacity. Prioritizes the left
-                output and can be used as an overflow gate.
+            name: Tempat Penyimpanan
+            description: Menyumpan bentuk yang berlebuhan, hingga kapasitas yang tertentu. Memprioritaskan output dari kiri
     wire_tunnel:
         default:
-            name: Wire Crossing
-            description: Allows to cross two wires without connecting them.
+            name: Penyebrangan Kabel
+            description: Memungkinkan untuk menyebrangkan 2 kabel tanpa menyambungkannya.
     constant_signal:
         default:
-            name: Constant Signal
-            description: Emits a constant signal, which can be either a shape, color or
-                boolean (1 / 0).
+            name: Sinyal Konstan
+            description: Mengeluarkan sinyal yang konstan, dapat berupa bentuk, warna atau boolean (1 / 0).
     lever:
         default:
-            name: Switch
-            description: Can be toggled to emit a boolean signal (1 / 0) on the wires layer,
-                which can then be used to control for example an item filter.
+            name: Saklar
+            description: Dapat diubah untuk mengeluarkan sinyal boolean (1 / 0) pada lapisan kabel,
+                yang bisa digunakan untuk mengontrol seperti penyaring.
     logic_gate:
         default:
-            name: AND Gate
-            description: Emits a boolean "1" if both inputs are truthy. (Truthy means shape,
-                color or boolean "1")
+            name: Gerbang AND
+            description: Mengeluarkan boolean "1" jika kedua input adalah benar. (Benar berarti sebuah bentuk,
+                warna atau boolean "1")
         not:
-            name: NOT Gate
-            description: Emits a boolean "1" if the input is not truthy. (Truthy means
-                shape, color or boolean "1")
+            name: Gerbang NOT
+            description: Mengeluarkan  boolean "1" jika input adalah tidak benar. (Benar berarti sebuah bentuk,
+                warna atau boolean "1")
         xor:
-            name: XOR Gate
-            description: Emits a boolean "1" if one of the inputs is truthy, but not both.
-                (Truthy means shape, color or boolean "1")
+            name: Gerbang XOR
+            description: , namun bukan keduanya.
```

Comment:
I don't speak malay, but it feels like something is missing.

It's also causing an error in the file, as well.